exploration/04_trigger_exploration.R

```diff
@@ -400,84 +400,84 @@ df_chirps_wy_2000 <- df_chirps |>
 
 # look at CHIRPS just across MAM
 
-df_chirps_mam_wy <- df_chirps |> 
+df_chirps_mam_wy <- df_chirps |>
   dplyr$mutate(
     month = lubridate$month(date),
     year = lubridate$year(date)
-  ) |> 
+  ) |>
   dplyr$filter(
     month %in% 3:5,
     year >= 1984
-  ) |> 
+  ) |>
   dplyr$group_by(
     year
-  ) |> 
+  ) |>
   dplyr$mutate(
     precipitation_cum = cumsum(value)
   ) |>
   dplyr$group_by(
     month
-  ) |> 
+  ) |>
   dplyr$arrange(
     precipitation_cum,
     .by_group = TRUE
-  ) |> 
+  ) |>
   dplyr$slice(
     1:8 # 1 in 5 year RP
-  ) |> 
-  dplyr$ungroup() |> 
+  ) |>
+  dplyr$ungroup() |>
   dplyr$transmute(
     indicator = "CHIRPS",
     month,
     year
   )
 
-df_emdat_wy <- df_emdat |> 
+df_emdat_wy <- df_emdat |>
   dplyr$transmute(
     indicator = "EMDAT",
-    year = c(2000, 2001, 2002, 2007, 2008, 2011, 2018, 2019, 2021, 2022, 2023),
+    year = c(2001, 2007, 2008, 2011, 2018, 2023),
```

Comment:
this years are basically taken from the start_year & end_year, but it looks like there was a bit of interpretation and perhaps country-level inputs?